server/src/test/resources/csars/yaml/valid/lamp-csar_NoInput/mysql_dbms/mysql_dbms_install.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
obsolete

Reply:
i think for the NoInput version we need this one because we set the pw for the database in this file